docs/project_management/new_projects.md

```diff
@@ -7,34 +7,35 @@ Note that if you are an external organization or external project, these guideli
 ## Forming a Team
 You can work by yourself, but managing an open source project by yourself can be very difficult. We highly recommend forming a team if you don't already have one!
 
-We have a special team forming process called [Project Project Pairing](membership/project_pairing), which occurs in the first two weeks of the semester.
+#### 1. Project Pitch
+You can find team members by word of mouth or by pitching your idea to the RCOS community. Read about Project Pitches [here](project_management/pitch.md).
 
-## Writing a Proposal
+#### 2. Project Pairing
 
-After you form a team, you will have to write a proposal for your project for the current semester. See the [rubric](grading/documentation?id=proposal) for more details on proposals. Make sure it meets the requirements specified, as mentors/Coordinators will be reviewing these and giving feedback before contributing starts.
+We have a special team forming process called [Project Project Pairing](membership/project_pairing), which occurs in the first two weeks of the semester.
 
 ## Setup
 
-If you haven't already, set up an account on:
-  - RCOS Discord
-  - Observatory
-  - GitHub
+#### 1. Writing a Proposal
 
-## Discord channel
-
-**You shouldn't have to do anything!** When projects are finalized, we automate the creation of categories for each small group which include a voice and text channel for your project. The text channel is only visible to you, your team, and the Coordinators/Faculty Advisors. You are given permission to pin and delete messages in your project's channel. Reach out to a Coordinator if you every have issues with your channel or need help. 
+After you form a team, you will have to write a proposal for your project for the current semester. See the [rubric](grading/documentation?id=proposal) for more details on proposals. Make sure it meets the requirements specified, as mentors/Coordinators will be reviewing these and giving feedback before contributing starts.
 
-## Setup Your Repository
+#### 2. Project Repository
 
-Create a repository on Github for your project. Then, create the following:
+Create a repository on GitHub (or similar) for your project. Then, create the following:
 - README
 - LICENSE
```

Comment:
Perhaps add some info about open source licences here?

Reply:
There's a short blurb about licenses if you click the link below there: https://handbook.rcos.io/#/grading/documentation

Do you think we should make it more explicit on the New Projects page in case they don't click that link?

Reply:
I think just having it mentioned below like it is (the "see documentation guide" part) is probably be fine.